project/src/main/webapp/login.jsp

```diff
@@ -2,32 +2,33 @@
 <%@ page import="com.google.sps.models.*" %>
 <!DOCTYPE html>
 <html>
-<head>
-   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-   <title>Log In Test</title>
-   <% User user = null;
-   if (session.getAttribute("userID") != null) {
-      user = Database.getUserByID((long) session.getAttribute("userID")); 
-   } %>
-</head>
-   
-<body onload="load()">
-  <p id="log"></p> 
-  
-  <% if (session.getAttribute("userID") != null) { %>
-  <p> <%= session.getAttribute("userID") %>, <%= user.getNickname() %> </p>
-  <% } %>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Log In Test</title>
+     <link rel="stylesheet" href="main.css" />
+    <% User user = null;
+    if (session.getAttribute("userID") != null) {
+        user = Database.getUserByID((long) session.getAttribute("userID")); 
+    } %>
+  </head>
+    
+  <body onload="load()">
+    <div id="log" style="position: absolute; top: 50%; left: 50%;"></div> 
+    
+    <% if (session.getAttribute("userID") != null) { %>
```

Comment:
this feels a little bit redundant with line 10; why not move what's happening in line 10 down here? Or we can also automatically redirect in line 10 if the userID doesn't exist, which guarantees the user exists when we reach this line.

Reply:
You are correct, I set it up this way so that later on I will be able to access this variable from anywhere in the page but as of now it is redundant. I am going to rework this entire page very soon so I will deal with this.